lychee-lib/src/extract/html/mod.rs

```diff
@@ -1,3 +1,4 @@
+//! Extract things from html documents
```

Comment:
```suggestion
//! Extract links and fragments from html documents
```

Did I miss anything in that comment? 🤔

Reply:
Nope looks good